CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(SKIP_GOOGLETEST_INSTALL "Skip installation" ${SKIP_GOOGLETEST_INSTALL} )
```

Comment:
also, slight extreme nitpick: `SKIP_GOOGLETEST_INSTALL` invokes negative logic. And you should not never not invoke negative logic!